sklearn/tests/test_calibration.py

```diff
@@ -156,27 +156,23 @@ def test_calibration_cv_splitter(data, ensemble):
 @pytest.mark.parametrize("method", ["sigmoid", "isotonic"])
 @pytest.mark.parametrize("ensemble", [True, False])
 def test_sample_weight(data, method, ensemble):
-    n_samples = 100
-    X, y = data
+    X, y = make_blobs((100, 1000), center_box=(-1, 1), random_state=42)
```

Comment:
Could you keep the previous test as-is and add a new test `test_sample_weight_class_imbalanced` with your new test.

Reply:
The previous test was very weak: just checking that ```sample_weight``` changes something without to the predictions without being more specific. I did not think it was worth keeping. But I understand that removing tests creates risks. I'll put it back.